### PR TITLE
Fix lookup endpoint with missing filed availableVersions

### PR DESCRIPTION
**Pull request by jbartece**

### CODE REVIEW

File: reports-backend/src/main/java/org/jboss/da/reports/backend/impl/VersionFinderImpl.java [hunk 1]
```diff
@@ -61,18 +64,30 @@ public String getBestMatchVersionFor(GAV gav, List<String> availableVersions) {
         return findBiggestMatchingVersion(gav, availableVersions);
     }
 
+    private List<String> getBuiltVerionsFor0(List<String> allVersions) {
```
Comment (thescouser89) [inline reports-backend/src/main/java/org/jboss/da/reports/backend/impl/VersionFinderImpl.java]: typo in method name, and it ends with '0'

File: reports-backend/src/main/java/org/jboss/da/reports/backend/api/VersionFinder.java [hunk 2]
```diff
@@ -13,13 +14,23 @@
 public interface VersionFinder {
 
     /**
-     * Finds all RedHat built artifacts (with suffix -redhat) with the same GA
+     * Finds all Red Hat built artifacts (with suffix -redhat) with the same GA
      * 
      * @param gav GroupId and ArtifactId, which specifies the artifact
-     * @return Found built RedHat artifacts with the same GA or null if the GA is not found
-     * @throws CommunicationException when there is a problem with communication with remote services.
+     * @return Found built RedHat artifacts with the same GA or null if the GA was not found
```
Comment (thescouser89) [inline reports-backend/src/main/java/org/jboss/da/reports/backend/api/VersionFinder.java]: s/RedHat/Red Hat

I guess we should do that change to be consistent.

**Comment by thescouser89** [comment]
So I've tried the cli with this PR and everything works fine!

So I guess fix the minor issues I found, then feel free to merge it!

File: reports-backend/src/main/java/org/jboss/da/reports/backend/impl/VersionFinderImpl.java (hunk 1 above)
Comment (jbartece) [inline reports-backend/src/main/java/org/jboss/da/reports/backend/impl/VersionFinderImpl.java]: The 0 is intentional. An extracted method, which contains main part of the logic

File: reports-backend/src/main/java/org/jboss/da/reports/backend/api/VersionFinder.java (hunk 2 above)
Comment (jbartece) [inline reports-backend/src/main/java/org/jboss/da/reports/backend/api/VersionFinder.java]: Yeah. I'll fix it

File: reports-backend/src/main/java/org/jboss/da/reports/backend/impl/VersionFinderImpl.java (hunk 1 above)
Comment (thescouser89) [inline reports-backend/src/main/java/org/jboss/da/reports/backend/impl/VersionFinderImpl.java]: Gotcha! Hopefully we won't have too many of these methods though! So I guess now just the typo in 'versions'

**Pull request merged**